### ISSUE TITLE
"Call to undefined method Illuminate\\Events\\Dispatcher::fire()"

### ISSUE THREAD

**Issue by BechirAhmed** [laravel-notification-channels/pusher-push-notifications#38]
I got this error, i don't know why, i use pusher via database and broadcast and it's works fine, but with this package it gives me this error
```
"Call to undefined method Illuminate\\Events\\Dispatcher::fire()"

**Comment by heathdutton** [comment]
This is because:
1) There was an error talking to Pusher. Likely invalid credentials. Try checking those and clearing cache.
2) The code has a bug w/ Laravel 5.8+ since it uses a method that's been deprecated, so the real error is buried under this exception.

**Comment by atymic** [comment]
Closed in #42
